hack/istio/multicluster/deploy-kiali.sh

```diff
@@ -81,6 +81,8 @@ deploy_kiali() {
         "--set deployment.image_version=dev"
       )
     elif [ "${KIALI_AUTH_STRATEGY}" == "openshift" ]; then
+      echo "Pushing the images into the cluster..."
+      make -e DORP="${DORP}" CLUSTER_TYPE="openshift" cluster-status
```

Comment:
Don't forget the `-C` like what is being done up on line 77 and down on line 89:

```suggestion
      make -e -C "${KIALI_REPO_ROOT}" DORP="${DORP}" CLUSTER_TYPE="openshift" cluster-status
```